### DISCUSSION TITLE
Running transmission-daemon.service as a regular user (kind of)

### DISCUSSION THREAD

**Discussion by pharaonic-faery** [transmission/transmission#5613] [Q&A]
Hello!

I don't know if what I'd like to do is possible or not, but here is the thing:
I'm running Transmission-daemon on a headless Raspberry Pi 4. I'd like it to run automatically at startup, so I enabled it with the `sudo systemctl enable transmission-daemon.service` command, rather than executing the simple `transmission-daemon` command each time I need it.

I'm not sure, but I think I understood that the "debian-transmission" user needs write access to the folders in which I want to download files via Transmission. Is it possible (and yet not unsecure) to edit some config files so that transmission-daemon.service take into account the write permissions of my regular user (UID 1000) instead of "debian-transmission" user's? (much like the 1000 UID user having executed the Transmission daemon)

I know I can use the `transmission-daemon` command as a workaround, but I have to run that command after each restart, and I would like this to be set once and for all without having the regular need of the CLI.
So, does it seem possible?
Thanks! :)

-----------------------------------------------
Rapsberry Pi 4 2GB
Raspberry Pi OS Bullseye (Debian 11)
transmission-daemon 3.00-1

**Comment by maxz** [comment]
Of course it's possible. But not in a sensible way by editing Transmission's configuration. You could do it by adjusting the Umask, but what would be rather insecure.

You could adjust the service file so that it runs as your regular user, but that would not be optimal either.

The best way would probably be to add your user to the `debian-transmission` group or to give read and optionally write permission on the directory to your user via ACL.
